extension/public/popup.css

```diff
@@ -258,6 +261,15 @@ header {
   background-color: rgba(255, 255, 255, 0);
 }
 
+/* Tag Item's Text */
+.keywords .tag-list .tag-item .tagText{
```

Comment:
nit pick: `tagText` --> `tag-text` (coding convention this repository has adopted)